io.github.ebonjaeger.bluejay.json

```diff
@@ -0,0 +1,58 @@
+{
+    "id": "io.github.ebonjaeger.bluejay",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.8",
+    "sdk": "org.kde.Sdk",
+    "command": "bluejay",
+    "finish-args": [
+        "--allow=bluetooth",
+        "--device=all",
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--system-talk-name=org.bluez"
+    ],
+
+    "modules": [
+        {
+            "name": "kirigami-addons",
+            "config-opts": [
+                "-DBUILD_TESTING=OFF",
+                "-DCMAKE_BUILD_TYPE=Release"
+            ],
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/kirigami-addons/kirigami-addons-1.7.0.tar.xz",
+                    "sha256": "7d854975185cf740de486d84f1e2ec3e4eadb66ac9fb897214cba9ab8cc492b1",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "project-id": 242933,
+                        "stable-only": true,
+                        "url-template": "https://download.kde.org/stable/kirigami-addons/kirigami-addons-$version.tar.xz"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "bluejay",
```

Comment:
https://github.com/EbonJaeger/bluejay/blob/df038b3089dec49bdfcdddab5f9192e6ab88eb10/io.github.ebonjaeger.bluejay.metainfo.xml#L62 this needs to be a direct link not to the html page. 

https://raw.githubusercontent.com/EbonJaeger/bluejay/refs/tags/v1.0.1/screenshot.png

Reply:
I see, thanks! This should be fixed in the latest release that I just pushed. This PR has also been updated to point to it.